src/utils/L.Map.Sync.js

```diff
@@ -0,0 +1,91 @@
+import L from 'leaflet'
+
+const noAnimate = { animate: false }
+
+L.Map.include({
+    _syncedMaps: {},
+    _onDragEndSync: {},
+
+    sync: function(id) {
+        const { setView } = L.Map.prototype
+        const origin = this
+        let syncedMaps = this._syncedMaps[id]
+
+        if (!syncedMaps) {
+            syncedMaps = []
+            this._syncedMaps[id] = syncedMaps
+        }
+
+        if (!syncedMaps.includes(this)) {
+            syncedMaps.push(this)
+
+            L.extend(this, {
+                setView: (center, zoom, options) =>
+                    syncedMaps.forEach(map => {
+                        setView.call(map, center, zoom, options)
+                    }),
+                _onResize: evt => console.log,
+            })
+
+            // Override method to drag all synced maps at the same time
+            this.dragging._draggable._updatePosition = function() {
+                const { getPosition, setPosition } = L.DomUtil
+                const oldPos = getPosition(this._element)
+
+                L.Draggable.prototype._updatePosition.call(this)
+
+                const newPos = getPosition(this._element)
+                const changePos = oldPos.subtract(newPos)
+
+                syncedMaps
+                    .filter(map => map !== origin)
```

Comment:
Minor: it would be a bit faster to just make this an `if` statement within the `forEach`, otherwise with `filter` there's another traversal of the array and a copy is made just to exclude one element.  i.e.

```js
syncedMaps.forEach(map => {
  if (map === origin) {
    return
  }
  ...
})
```

Reply:
Changed!